latency.go

```diff
@@ -236,9 +240,17 @@ func main() {
 	log.Println("==============================")
 
 	if HistFile != "" {
-		pctls := histwriter.Percentiles{10, 25, 50, 75, 90, 99, 99.9, 99.99, 99.999}
+		pctls := histwriter.Percentiles{10, 25, 50, 75, 90, 99, 99.9, 99.99, 99.999, 99.9999, 99.99999, 100.0}
```

Comment:
Do we not do these at all to stdout anymore?

Reply:
Not all, originally stdout stopped at 99.99.  I'll add all values.

Reply:
I updated the output.  It made sense to adjust the formatting algorithm, although it is personal preference - open to other suggestions.